crypto-ffi/src/CoreCrypto.udl

```diff
@@ -73,6 +73,7 @@ enum CryptoError {
     "UnauthorizedExternalCommit",
     "InvalidHashReference",
     "GenerationOutOfBound",
+    "DecryptionError",
```

Comment:
Wouldn't it make sense to make the name more specific given that it only happen when local epoch is out sync?

Reply:
Ah too slow, already merged